src/cart/src/include/gurt/fault_inject.h

```diff
@@ -129,13 +129,17 @@ int d_fault_inject_fini(void);
 
 /**
  * Start injecting faults.
+ *
+ * \return                   DER_SUCCESS on success, DER_NOSYS if not supported
  */
-void d_fault_inject_enable(void);
+int d_fault_inject_enable(void);
```

Comment:
copyright date in header needs an update

Reply:
Updated.